au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/TrackClipsItem.qml

```diff
@@ -197,11 +205,28 @@ Item {
         }
     }
 
+    ChannelSplitter {
+        id: channelSplitter
+
+        anchors.fill: parent
+        anchors.topMargin: trackViewState.isTrackCollapsed ? 1 : 21
+        anchors.bottomMargin: 3
```

Comment:
if some known values ​​are used in calculations then please use them
there are many magic numbers here - 1, 21, 3, 0.05

Reply:
I would love to. Could you help me with getting clip inner margin values?

Reply:
@Eism, the problem here is that we need to align Track's splitter with Clip's splitter. The padding and margin of the Clip are not accessible (or I don't know how to access them).
As for the styles, colors, and opacity, let me know if you prefer them to be moved to theme consts